t/mojo/daemon.t

```diff
@@ -18,13 +18,21 @@ use Mojolicious;
 package TestApp;
 use Mojo::Base 'Mojolicious';
 
+my $message = 'Test';
+
 sub handler {
   my ($self, $tx) = @_;
   $tx->res->code(200);
-  $tx->res->body('Hello TestApp!');
+  $tx->res->body("Hello $message!");
   $tx->resume;
 }
 
+# Ensure the 'startup' routine it called
+sub startup {
+	my $self = shift;
+	$message = 'TestApp';
```

Comment:
Not run through perltidy. Either way, I'm not sure if this is testing what you want.

Reply:
I adjusted indentation afterwards.
This is not checking the hook indeed, I can remove this test.